src/main/java/sparrow/data/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package sparrow.data.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Deadline in the task list.
+ */
+public class Deadline extends Task {
+    protected LocalDate dueDate;
+
+    /**
+     * Creates a Deadline.
+     * @param description details of deadline
```

Comment:
I think you can add a line in between the parameters and the description of the method